frog/imports/ui/TeacherView.jsx

```diff
@@ -83,25 +85,49 @@ const SessionController = ( { session, activities } ) => {
   )
 }
 
-const SessionList = ( { sessions } ) => { return(
-  <div>
-    <h3>Session list</h3>
-    <ul> { 
-      sessions.map((session) => 
-        <li key={session._id}>
-          <a href='#' onClick={() => Sessions.remove({_id:session._id})}>
-            <i className="fa fa-times"></i>&nbsp;
-          </a>
-          <a href='#' onClick={ () => setTeacherSession(session._id) }>
-            <i className="fa fa-pencil"></i>&nbsp;
-          </a>&nbsp;
-          {session._id}
-        </li>
-      ) 
-    } </ul>
-    <button className='btn btn-primary btn-sm' onClick={addSession}>Add session</button>
-  </div>
-)}
+const SessionControllerContainer = createContainer(({ session }) => {
+  return({
+    session: session,
+    activities: session? Activities.find({graphId: session.graphId}).fetch() :null
+  })
+}, SessionController)
+
+const SessionList = ( { sessions, graphs } ) => { 
+  var selectedGraph = graphs[0] ? graphs[0]._id: null
```

Comment:
If the list of graphs is empty graph[0] is undefined, so it crashes on graphs[0]._id